media/css/base/svg-animation-check.less

```diff
@@ -0,0 +1,37 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#svg-test {
+    visibility: hidden;
+    height: 0;
+    width: 0;
+    overflow: hidden;
+}
+
+#svg-test-circle {
+    stroke-dasharray: 117;
+    stroke-dashoffset: 117;
+    -webkit-animation: svg-dash-test 0.1s linear;
+    -webkit-animation-fill-mode: forwards;
+    animation: svg-dash-test 0.1s linear;
+    animation-fill-mode: forwards;
```

Comment:
Small perf nit - could consolidate and put `forwards` in `animation` declaration above?